lib/sagacious_succotash/app_builder.rb

```diff
@@ -60,6 +60,29 @@ def configure_simple_form
       generate "simple_form:install"
     end
 
+    def pundit_generate
+      generate "pundit:install"
+    end
+
+    def pundit_add_after_actions_to_application_controller
+      code = <<-CODE.strip_heredoc.prepend("  ")
+        # Enable Pundit
+          include Pundit
```

Comment:
Haha yeah I kind of agree. This seemed like the lesser of the evils...One reason I added the tests is so if I come up with a better way to write this (I want to write an indent_block method or something), it can be refactored easily.